### ISSUE TITLE
Capabilities API endpoint

### ISSUE THREAD

**Issue by olemis** [exchangeproject/exchange#23]
[GET] /api/capabilities

Should return API capabilities set. Each optional operation has corresponding flag in the capabilities. Optional operations should be implemented if particular blockchain provides such functionality.

Response:

```js
{
// Should be  true,  if  [PUT] /api/transactions call
// is supported
“isTransactionsRebuildingSupported”: boolean,
// Should be  true  if
//  [POST] /api/transactions/many-inputs and
//  [GET] /api/transactions/broadcasted/many-inputs calls
// are supported
“areManyInputsSupported”: boolean,
 // Should be t rue  if
//  [POST] /api/transactions/many-outputs and
//  [GET] /api/transactions/broadcasted/many-outputs calls
// are supported
“areManyOutputsSupported”: boolean
}
```

**Comment by stdevAlek6** [comment]
already implemented

**Comment by olemis** [comment]
closed by #29

**Comment by stdevAlek6** [comment]
Finished
